db/seeds.rb

```diff
@@ -1,8 +1,113 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-
-james = User.where(email: 'jkerr@chapman.edu').first_or_initialize
-james.first_name = 'James'
-james.last_name  = 'Kerr'
-james.role = :super_admin
-james.save
+# This file should contain all the record creation needed to seed the database with its default
```

Comment:
A good seed can really ease development. Usually worth the investment to sort out a minimal default dataset.

Can be augmented with `rake` tasks to generate test data.